src/cephadm/cephadm.py

```diff
@@ -7463,14 +7508,20 @@ def get_ceph_cluster_count(ctx: CephadmContext) -> int:
     return len([c for c in os.listdir(ctx.data_dir) if is_fsid(c)])
 
 
-def command_rm_cluster(ctx):
-    # type: (CephadmContext) -> None
+def command_rm_cluster(ctx: CephadmContext) -> None:
     if not ctx.force:
         raise Error('must pass --force to proceed: '
                     'this command may destroy precious data!')
 
     lock = FileLock(ctx, ctx.fsid)
     lock.acquire()
+    _command_rm_cluster(ctx, ctx.keep_logs, ctx.zap_osds)
+
+
+def _command_rm_cluster(ctx: CephadmContext, keep_logs: bool, zap_osds: bool) -> None:
```

Comment:
My last comment about renaming this function got marked resolved but the function name seems to still be the same. Am I missing something or did you decide you don't want to change the name?

Reply:
upss I think I just forgot to rename it. Done.